pkg/apis/extensions/install/install_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	"testing"
 
 	"k8s.io/kubernetes/pkg/api"
+	_ "k8s.io/kubernetes/pkg/api/install"
```

Comment:
Removing. (this is the sort of change that adding things directly to api.Scheme in the api/register.go papers over.)